src/operations/blas3/gemm_local.hpp

```diff
@@ -278,12 +278,9 @@ class Gemm<input_t, output_t, DoubleBuffer, NbcA, NbcB, ClSize, TileType,
     const index_t b_size = trans_b ? ldb * k : n * ldb;
     const index_t c_size = ldc * n;
 
-    auto ptr_A = a_.get_data().get_pointer() + a_.get_access_displacement() +
-                 (wg_batch_id * stridea_);
-    auto ptr_B = b_.get_data().get_pointer() + b_.get_access_displacement() +
-                 (wg_batch_id * strideb_);
-    auto ptr_C = c_.get_data().get_pointer() + c_.get_access_displacement() +
-                 (wg_batch_id * stridec_);
+    auto ptr_A = a_.get_pointer() + (wg_batch_id * stridea_);
```

Comment:
Thanks for your PR, @muhammad-tanvir-1211.

Why did you remove the `get_access_displacement`? I think it should be available also for USM.

Reply:
`get_access_displacement` is not needed for the USM. It has been added because buffer could not handle capturing arithmetic operation on the device side from the host side.

Reply:
yes